docs/user-guide/command-line-interface.md

```diff
@@ -4,23 +4,23 @@ To run ESLint on Node.js, you must have npm installed. If npm is not installed,
 
 Once npm is installed, run the following
 
-    npm i -g eslint
+    npm i eslint --save-dev
 
 This installs the ESLint CLI from the npm repository. To run ESLint, use the following format:
 
-    eslint [options] [file|dir|glob]*
+    npx eslint [options] [file|dir|glob]*
```

Comment:
I was wondering if we could add a note to say how to use the local-installed eslint, e.g. `npx/yarn/pnpm`, instead of putting `npx` before every eslint command.

Reply:
I think for beginners it will be confusing to remove `npx` everywhere. I agree it's not pretty, but given that people will probably copy and paste from this page, I'd rather give them the complete command.